test/actions/FilterActionsTest.js

```diff
@@ -0,0 +1,180 @@
+'use strict'
+import { expect } from 'chai'
+import queryString from 'query-string'
+import * as filterActions from '../../src/actions/filterActions'
+
+describe('FilterActions', function () {
+    describe('getFilterFromUrl', function() {
+        const testQueryOne = '?eventTypes=16&eventTypes=11'
+        const expectedValue = [16, 11] 
+        const resultOne = filterActions.getFilterFromUrl(testQueryOne)
+        it('should return the right value from the query string', function() {
+            expect(resultOne).to.be.an('object')
+            expect(resultOne.eventTypes).to.be.an('array')
+            expect(resultOne.eventTypes[0]).to.equal(expectedValue[0])
+            expect(resultOne.eventTypes[1]).to.equal(expectedValue[1])
+        })
+        it('should return an array when query string includes a single filter value', function() {
+            const testQueryTwo = '?eventTypes=1'
+            const resultTwo = filterActions.getFilterFromUrl(testQueryTwo)
+
+            expect(resultTwo).to.be.an('object')
+            expect(resultTwo.eventTypes).to.be.an('array')
+        })
+        it('should return an int array', function() {
+            const unexpectedValue = ["16", "11"]
+
+            expect(resultOne.eventTypes[0]).to.not.equal(unexpectedValue[0])
```

Comment:
pseudocode expectations:

typeof(resultOne.eventTypes[0]) is a Integer
expect(resultOne.eventTypes[0]).to.equal(16)